server/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+class UsersController < ApplicationController
+  def index
```

Comment:
I agree that we don't need these browser views.

Reply:
My understanding is that we will still have an Application Controller, regardless of whether we are creating something for the browser of an API.

Reply:
*or an API

Reply:
you are correct, a controller is needed in any case. From what I gathered from the article I posted above, "the ApplicationController [should] inherit from ActionController::API instead of ActionController::Base"